src/sample/sample.fxml

```diff
@@ -1,21 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.chart.CategoryAxis?>
-<?import javafx.scene.chart.LineChart?>
-<?import javafx.scene.chart.NumberAxis?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.control.Menu?>
-<?import javafx.scene.control.MenuBar?>
-<?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.Slider?>
-<?import javafx.scene.control.SplitPane?>
-<?import javafx.scene.control.Tooltip?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.text.Font?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
 
 <AnchorPane fx:id="mainAnchorPane" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="600.0" prefWidth="900.0" style="-fx-background-color: #8F8F8F; -fx-border-color: #8F8F8F;" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="sample.Controller">
     <children>
-        <Button fx:id="runBtn" layoutX="140.0" layoutY="477.0" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" onMouseClicked="#onRunClicked" prefHeight="50.0" prefWidth="150.0" style="-fx-background-color: #68a2f4;" text="RUN" textFill="WHITE" AnchorPane.leftAnchor="140.0" AnchorPane.rightAnchor="140.0" AnchorPane.topAnchor="477.0" />
+        <Button fx:id="gatherBtn" layoutX="609.0" layoutY="510.0" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" onMouseClicked="#onGatherClicked" prefHeight="50.0" prefWidth="253.0" style="-fx-background-color: #68a2f4;" text="Gather mode" textFill="WHITE" AnchorPane.leftAnchor="609.0" AnchorPane.rightAnchor="40.0" AnchorPane.topAnchor="510.0" />
+
+        <Button fx:id="speakBtn" layoutX="611.0" layoutY="466.0" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" onMousePressed="#onSpeakPressed" prefHeight="30.0" prefWidth="249.0" style="-fx-background-color: RED;" text="Hold to Speak" textFill="WHITE" AnchorPane.leftAnchor="611.0" AnchorPane.rightAnchor="42.0" AnchorPane.topAnchor="466.0" />
+
+        <Label fx:id="JolinnaLabel" layoutX="52.0" layoutY="7.0" prefHeight="30.0" prefWidth="212.0" text="Welcome to Walk and Talk" textFill="BLACK">
+         <font>
+            <Font size="16.0" />
+         </font></Label>
```

Comment:
All the code for this label should go on one line just like the buttons.  There is also no corresponding label in the controller file with the id JolinnaLabel.  I suggest changing this to welcomeLbl.